corehq/apps/sms/tasks.py

```diff
@@ -162,8 +162,10 @@ def handle_domain_specific_delays(msg, domain_object, utcnow):
 
 
 def message_is_stale(msg, utcnow):
-    oldest_allowable_datetime = \
-        utcnow - timedelta(hours=settings.SMS_QUEUE_STALE_MESSAGE_DURATION)
+    sms_stale_duration = msg.custom_metadata.get('sms_stale_after', None) if msg.custom_metadata else None
+    sms_stale_duration = sms_stale_duration \
```

Comment:
Couldn't this be shortened to `sms_stale_duration or settings.SMS_QUEUE_STALE_MESSAGE_DURATION`?

Reply:
`sms_stale_duration` can be 0 so I think we cannot use this here.

Reply:
Wouldn't a duration of 0 mean no messages ever get sent?

Reply:
Yeah. Correct 😅 
The minimum value should not be zero. I will update it.

Reply:
Yeah. Correct.
I will update the min value to 1.
I don't know what I was thinking while keeping it 0 😅